tests/models/test_player.py

```diff
@@ -0,0 +1,23 @@
+"""A module for testing player model."""
+import pytest
+from django.contrib.auth.models import User
+
+from api.models import Player
+
+pytestmark = pytest.mark.django_db
+
+
+def test_player_is_created():
+    """Test if Player and User is created."""
+    user = User.objects.create_user("my_username", "email@email.it", "password")
+    player = Player(user=user, nickname="my_nickname", motto="short motto")
+    user.save()
+    player.save()
+
+    db_user = User.objects.get(username="my_username")
```

Comment:
Try swapping `User` and `Player` - and instead of asserting user's properties, simply check whether the obtained player has the `user` field not empty.